tests/test_utils.py

```diff
@@ -53,13 +52,6 @@ def test_time_normalize(self):
             normalized = normalize_timing_and_sort(events)
             self.assert_sort(normalized, date_format)
 
-    def test_event_class_normalization(self):
```

Comment:
We probably want to keep this test class right? Just renaming to not have RTA specific language?

Reply:
I brought it back!